tests/slack_sdk_async/socket_mode/test_interactions_websockets.py

```diff
@@ -90,7 +90,9 @@ async def socket_mode_listener(
             expected.sort()
 
             count = 0
-            while count < 10 and len(received_messages) < len(expected):
+            while count < 10 and (
+                len(received_messages) < len(expected) or len(received_socket_mode_requests) < len(socket_mode_envelopes)
```

Comment:
good catch!

Reply:
Yeah! It took me well over an hour to figure out what's going on with PyPy (and CPython from time to time) and complete this PR 😅